python/pyspark/sql/readwriter.py

```diff
@@ -258,64 +283,73 @@ def parquet(self, *paths):
 
     @ignore_unicode_prefix
     @since(1.6)
-    def text(self, paths):
+    def text(self, path):
```

Comment:
I see. I should not change this like this. Let me revert this because someone might be using this by a named argument.